piptools/writer.py

```diff
@@ -65,6 +67,18 @@ def write_trusted_hosts(self):
             for trusted_host in dedup(self.trusted_hosts):
                 yield '--trusted-host {}'.format(trusted_host)
 
+    def write_find_links(self):
+        yielded = set()
+        dst_dir = os.path.dirname(self.dst_file)
+        for abs_link in self.find_links:
+            rel_link = os.path.relpath(abs_link, start=dst_dir)
+            if not rel_link.startswith(os.path.pardir + os.path.sep):
```

Comment:
Let's assume that user has `/app/stuff/requirements.txt` and `/app/links/`. In that case relative link would be `../links` and this link will be omitted. Do you think it's okay?